src/sentry/auth/superuser.py

```diff
@@ -75,6 +94,10 @@ def __init__(self, request, allowed_ips=UNSET, org_id=UNSET, current_datetime=No
 
     @property
     def is_active(self):
+        # if accessing differnt org. Also can't use self.org_id as ORG_ID is None
+        org = getattr(self.request, "organization", None)
+        if org and org not in self.request.user.get_orgs():
+            return self._check_expired_on_org_change()
```

Comment:
can we put this in a separate PR? seems like a separate concern of our logging we want to do.

Reply:
yup will do!